docs/maintain-guides-how-to-create-offline-transaction.md

```diff
@@ -0,0 +1,77 @@
+---
+id: maintain-guides-how-to-create-offline-transaction
+title: Generate an offline transaction
+sidebar_label: Generate an offline transaction
+---
+
+# Generating an offline transaction
+
+There are few different ways to create an offline transaction by using the [@polkadot@cli-signer](https://github.com/polkadot-js/tools/tree/master/packages/signer-cli), [txwrapper](https://github.com/paritytech/txwrapper) or [Subkey](https://github.com/paritytech/substrate/tree/master/bin/utils/subkey). This guide shows you how to use each one of these for different scenarios. 
+
+## polkadot@cli-signer
+
+This would be the most easy to use if you are looking for CLI approach and have some basic understanding of the public-key cryptography. You can use that to create any transaction and submit it(Extrinsic) to the network directly or you create an offline transaction and then submit it over the RPC or [PolkadotJS UI](https://polkadot.js.org/apps/#/toolbox).
```

Comment:
```suggestion
This would be the easiest to use if you are looking for a CLI approach and have some basic understanding of public-key cryptography. You can use that to create any transaction and submit it (Extrinsic) to the network directly or you create an offline transaction and then submit it over RPC or [PolkadotJS UI](https://polkadot.js.org/apps/#/toolbox).
```